spaceship/lib/spaceship/connect_api/models/app.rb

```diff
@@ -54,12 +54,12 @@ module EducationDiscountType
         "contentRightsDeclaration" => "content_rights_declaration",
 
         "appStoreVersions" => "app_store_versions",
+        # This attribute is already deprecated. It will be removed in a future release.
         "prices" => "prices"
```

Comment:
I have no idea whether we can drop this attribute.

We have to support a latest API

Reply:
As I see, old applications continue working with old attributes, so it is better to leave it, for backward compatibility.

Reply:
I disagree, this attribute should be removed as with this PR's changes it will always end up being `nil`. Part of the code seeing the `nil` might think this app has no price even though it might have prices set up in App Store Connect, ending up with confusion like in #21213.
Not having the attribute would catch possible misuses.

Reply:
Sorry, I might have misunderstood part of the code, as it seems app data can come from different locations, some with or without prices. The current state is pretty confusing 😅